wqu_app/message.py

```diff
@@ -29,10 +30,11 @@ def get_weather(coords):
 
 
 def greet(ip_address):
-    coords = get_geolocation(ip_address)
+    coords = get_geolocation(ip_address)[0]
     temperature = get_weather(coords)
+    city = get_geolocation(ip_address)[1]
```

Comment:
Notice how we are calling `get_geolocation` function twice? We can combine those two calls into one.